pkg/storage/azureblob/checker.go

```diff
@@ -0,0 +1,20 @@
+package azureblob
+
+import (
+	"context"
+
+	"github.com/gomods/athens/pkg/config"
+	"github.com/gomods/athens/pkg/errors"
+	"github.com/gomods/athens/pkg/observ"
+)
+
+// Exists implements the (./pkg/storage).Checker interface
+// returning true if the module at version exists in storage
+func (s *Storage) Exists(ctx context.Context, module string, version string) (bool, error) {
+	const op errors.Op = "azureblob.newBlobStoreClient.Save"
```

Comment:
i think we misunderstood again
the convention is `package.method` so in this case it was correct `azureblob.Save`

Reply:
Sorry my bad.